src/EditModePlayerManager.cpp

```diff
@@ -10,7 +10,13 @@
 
 
 void EditModePlayerManager::AddPlayers(const NoteData& note_data) {
-	FOREACH_EnabledPlayer(pn){
+	// In case of routine mode, both players are actually seen as enabled. This
```

Comment:
Can we move this comment down one line above the condition?

Reply:
Sure thing. Done.